torchao/quantization/quant_primitives.py

```diff
@@ -64,6 +64,18 @@ class ZeroPointDomain(Enum):
     INT = auto()
     FLOAT = auto()
 
+class TorchSignedInt(Enum):
```

Comment:
this can be more general since we may need some other things as well, like https://github.com/pytorch/ao/blob/52775076f833e7301fbac29eb2f4938461f1c94c/torchao/prototype/mx_formats/constants.py#L6-L8

so maybe just `TORCHAO_DTYPE` or something like that

Reply:
I added a `TorchAODType` enum with fields like INT1 to INT7. What do you think?

Reply:
yeah sounds good